src/libslic3r/ExtrusionEntity.hpp

```diff
@@ -301,6 +301,42 @@ class ExtrusionPath : public ExtrusionEntity
     bool m_no_extrusion = false;
 };
 
+class ExtrusionPathSloped : public ExtrusionPath
```

Comment:
Would it be better to make it derived from ExtrusionLoop?

Reply:
No, mainly beccause `GCode::_extrude` accepts an `ExtrusionPath` not `ExtrusionLoop`, where the actual extrusion gcodes are generated.